localstack/cli/localstack.py

```diff
@@ -617,15 +618,13 @@ def cmd_ssh() -> None:
     `MAIN_CONTAINER_NAME`.
     """
     from localstack.utils.docker_utils import DOCKER_CLIENT
-    from localstack.utils.run import run
 
     if not DOCKER_CLIENT.is_container_running(config.MAIN_CONTAINER_NAME):
         raise CLIError(
             f'Expected a running LocalStack container named "{config.MAIN_CONTAINER_NAME}", but found none'
         )
     try:
-        process = run("docker exec -it %s bash" % config.MAIN_CONTAINER_NAME, tty=True)
-        process.wait()
+        run_interactive(["docker", "exec", "-it", config.MAIN_CONTAINER_NAME, "bash"])
```

Comment:
I would argue that we do not need run_interactive here. It would only be necessary when we want to do something afterwards.

In this case, I would suggest using `os.execv*` here, as replacing the current process with the new one makes sure every signal is handled as intended.

Reply:
fair point. i don't really understand `os.exec` well and how it works on other platforms, but works fine on my machine this way :+1:

Reply:
It replaces the current process image with a new process image. Instead of running two processes, we only run one, and all signals will therefore get directly to docker, without any logic on our side :)

Reply:
Should be available on unix and windows :)